charts/newrelic-infrastructure/templates/controlplane/_tolerations_helper.tpl

```diff
@@ -0,0 +1,25 @@
+{{- /*
+Defaults for controlPlane while keeping then overridable.
```

Comment:
'them" ?

Reply:
This helper does not exist anymore. Resolving.